src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -140,6 +142,10 @@ public static class Configuration
 
 	// The target language of the generated client code. The value must be set to "CSharp" or "VB".
 	public const string TargetLanguage = "OutputLanguage";
+	

```

Comment:
TempFilePath related codes are not my changes. It's introduced from running "The Custom Tool" for the tt file.